release-notes/8.0/supported-os.json

```diff
@@ -131,11 +131,16 @@
                         "Arm64",
                         "ppc64le",
                         "s390x",
-                        
                         "x64"
                     ],
                     "supported-versions": [
                         "9"
+                    ],
+                    "exceptional-versions": [
+                        {
+                            "version": "8",
+                            "note": "CentOS Stream 8 was explicitly never supported."
+                        }
```

Comment:
@omajid From https://github.com/dotnet/core/pull/9382 I am not fully sure why CentOS Stream 8 was not officially supported. Maybe you want to help here with giving a better reason besides what I was able to come up with.

Reply:
We have a policy to not support an OS if it will go EOL within 6 months (or so) after a given .NET version ships. That rule is at play here.

https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/